src/lib/prompts.js

```diff
@@ -6,7 +6,7 @@ exports.generatePrompts = (versionList, selectOnly = false) => [
   {
     type: 'select',
     name: 'option',
-    message: 'Would you like to use an existing version or create a new one to associate with your OAS file?',
```

Comment:
We were previously using this language in a bunch of places (version commands, docs commands, etc.) 😬